src/data/factData.ts

```diff
@@ -0,0 +1,154 @@
+        factImage: "/images/insight.jpg",
+      },
+      {
+        factTitle: "Ingenuity",
+        factText:
+          "The Mars Helicopter, Ingenuity, is a technology demonstration to test powered, controlled flight on another world for the first time. It hitched a ride to Mars on the Perseverance rover. Once the rover reached a suitable 'airfield' location, it released Ingenuity to the surface so it could perform a series of test flights over a 30-Martian-day experimental window. Ingenuity has paved the way for future aerial missions on the planet and has been utlised in scouting areas of potential significance for the Perseverance rover to explore.",
+        factImage: "/images/ingenuity.jpg",
+      },
+    ],
+  },
+  {
+    sectionTitle: "Orbits, Rotation and Seasons",
+    facts: [
+      {
+        factTitle: "Orbital Period",
+        factText:
+          "As Mars orbits the Sun, it completes one rotation every 24.6 hours, which is very similar to one day on Earth (23.9 hours). Martian days are called sols, short for 'solar day'. A year on Mars lasts 669.6 sols, which is the same as 687 Earth days.",
+        factImage: "/images/marsyear.jpg",
+      },
+      {
+        factTitle: "That's A Long Commute",
+        factText:
+          "From an average distance of 142 million miles (228 million kilometers), Mars is 1.5 astronomical units away from the Sun. One astronomical unit (abbreviated as AU), is the distance from the Sun to Earth. From this distance, it takes sunlight 13 minutes to travel from the Sun to Mars.",
+        factImage: "/images/marssolarsystem.jpg",
+      },
+      {
+        factTitle: "Jaunty Angles",
+        factText:
+          "Mars' axis of rotation is tilted 25 degrees with respect to the plane of its orbit around the Sun. This is another similarity with Earth, which has an axial tilt of 23.4 degrees. Like Earth, Mars has distinct seasons, but they last longer than seasons here on Earth since Mars takes longer to orbit the Sun (because it's farther away). And while here on Earth the seasons are evenly spread over the year, lasting 3 months (or one quarter of a year), on Mars the seasons vary in length because of Mars' elliptical (egg-shaped) orbit around the Sun.",
+        factImage: "/images/marsseasons.webp",
+      },
+      {
+        factTitle: "Seasonal Changes",
+        factText:
+          "Spring in the northern hemisphere (autumn in the southern) is the longest season at 194 sols. Autumn in the northern hemisphere (spring in the southern) is the shortest at 142 days. Northern winter/southern summer is 154 sols, and northern summer/southern winter is 178 sols.",
+        factImage: "/images/marsseasons2.png",
+      },
+    ],
+  },
+];
+
+// export const getFacts = (): Section[] => {
+//   return facts
+// };
+
+export const getFacts = () => {
+  return facts;
+};
```

Comment:
If you're not randomising, you don't need this, you can just export the list above

Reply:
list exported at definition;